src/Effectful/Log.hs

```diff
@@ -35,14 +36,18 @@ runLog
   -- ^ The logging back-end to use.
   -> LogLevel
   -- ^ The maximum log level allowed to be logged.
+  -> [Text]
+  -- ^ The current application domain.
+  -> [Pair]
+  -- ^ Additional data to be merged with the log message's data
   -> Eff (Log : es) a
   -- ^ The computation to run.
   -> Eff es a
-runLog component logger maxLogLevel = evalStaticRep $ Log LoggerEnv
+runLog component logger maxLogLevel domain additionalData = evalStaticRep $ Log LoggerEnv
```

Comment:
Alternatively we could keep `runLog` as it is and provide an alternative interpreter `runLogWithEnv` (or some such name) that would take `LoggerEvn` on input.  The meaning of the arguments would be then clearer at the call site.  What do you think?

Reply:
yup, it's better this way :+1: